### PR TITLE
feat(Healthcare): Capacity for Service Unit, concurrent appointments based on Capacity, Patient enhancements

### PR DESCRIPTION
**Pull request by akurungadam**
1. Healthcare Service Unit Type
- Allow Overlap check
healthcare-service-unit-type

2. Healthcare Service Unit
- If Allow Overlap enabled, set Service Unit Capacity
healthcare-service-unit-capacity

3. Patient Appointment
- Book concurrent Appointments for overlap enabled service units within the capacity
check_availability

4. Patient Appointment
- Added tests to cover overlap

5. Healthcare Service Unit Tree
- Tree View fixes
- Option to create multiple Service Units
service-unit-add-multiple

6. Patient
- Address and Contact (also linked to associated customer)
- Dashboard
patient v2

`no-docs` Docs added in [wiki](https://docs.erpnext.com/docs/v13/user/manual/en/healthcare/healthcare_service_unit/edit?wiki_page_patch=9f57389e76)

### CODE REVIEW

**Comment by chechani** [comment]
It's a very interesting feature based on this PR and might be useful where appointments are based on tokens and not time slots.

**Comment by mujeerhashmi** [comment]
@ruchamahabal @akurungadam Please can we get this merged. Patient Address and Contact is a must required feature. Things are slowing down for Healthcare.

**Comment by mujeerhashmi** [comment]
Bump

**Comment by mujeerhashmi** [comment]
Keep Alive

File: erpnext/healthcare/doctype/healthcare_service_unit_type/healthcare_service_unit_type.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "service_unit_type",
   "allow_appointments",
   "overlap_appointments",
+  "allow_procedures",
```
Comment (ruchamahabal) [inline erpnext/healthcare/doctype/healthcare_service_unit_type/healthcare_service_unit_type.json]: This field has no code-level implementation. Can we remove this for now?

**Comment by ruchamahabal** [comment]
Changed the design for Available Slots dialog a bit:

photo_2021-08-19 18 12 44

**Review by ruchamahabal — APPROVED** [review]

**Pull request merged**